### PR TITLE
dotCMS/core#23554 Can't close query dialog on content search screen

### PR DESCRIPTION
**Pull request by zulqarnainvd**
### Proposed Changes
* Add close icon

### Checklist
- [ ] Tests
- [ ] Translations
- [ ] Security Implications Contemplated (add notes if applicable)

### Additional Info
** any additional useful context or info **

### Screenshots
Original             |  Updated
:-------------------------:|:-------------------------:
** original screenshot **  |  ** updated screenshot **
image
image

### CODE REVIEW

File: dotCMS/src/main/webapp/html/css/dijit-dotcms/dotcms.css [hunk 1]
```diff
@@ -11031,7 +11031,7 @@ a.category_higlighted, a.tag_higlighted {
 }
 
 .content-search__show-query-dialog .dijitDialogTitleBar {
-  height: 0px;
+  height: 18px !important;
```
Comment (fmontes) [inline dotCMS/src/main/webapp/html/css/dijit-dotcms/dotcms.css:11034]: Please don't use `!important;` unless is the last resource.

File: dotCMS/src/main/webapp/html/portlet/ext/contentlet/view_contentlets.jsp [hunk 2]
```diff
@@ -804,7 +804,7 @@
         <!-- END Right Column -->
 
         <!-- START Show Query -->
-        <div id="queryDiv" dojoType="dijit.Dialog" class="content-search__show-query-dialog" style="display: none;padding-top:15px\9;">
+        <div id="queryDiv" dojoType="dijit.Dialog" class="content-search__show-query-dialog noDijitDialogTitleBar" style="display: none;padding-top:15px\9;">
```
Comment (fmontes) [inline dotCMS/src/main/webapp/html/portlet/ext/contentlet/view_contentlets.jsp:807]: The `noDijitDialogTitleBar` is a class that dojo (or dijit) add when there is a title bar in the dialog component. In this case I think we can do this by adding a `title=""` to the dialog so it shows the title bar and the close button.

You can always search the dijit documentation and see what properties the widgets takes, for example here is for the dialog: https://dojotoolkit.org/reference-guide/1.6/dijit/Dialog.html

**Review by fmontes — CHANGES_REQUESTED** [review]

File: dotCMS/src/main/webapp/html/css/dijit-dotcms/dotcms.css (hunk 1 above)
Comment (zulqarnainvd) [reply dotCMS/src/main/webapp/html/css/dijit-dotcms/dotcms.css:11034]: dijit dialog adding height dynamically on a height that's why I used important.

Comment (rjvelazco) [reply dotCMS/src/main/webapp/html/css/dijit-dotcms/dotcms.css:11034]: Maybe adding the `height` inline? We can try @zulqarnainvd

**Review by rjvelazco — APPROVED** [review]

File: dotCMS/src/main/webapp/html/css/dijit-dotcms/dotcms.css (hunk 1 above)
Comment (zulqarnainvd) [reply dotCMS/src/main/webapp/html/css/dijit-dotcms/dotcms.css:11034]: dijit dialog adding a title bar dynamically so we don't have access to this element or we can use js for adding height.

**Pull request merged**